Lib/fontbakery/profiles/opentype/__init__.py

```diff
@@ -1,30 +1,30 @@
 from fontbakery.fonts_profile import profile_factory
 from fontbakery.section import Section
 
+from .cff import *
+from .cmap import *
+from .head import *
+from .os2 import *
+from .post import *
+from .name import *
+from .loca import *
+from .hhea import *
+from .dsig import *
+from .gdef import *
+from .gpos import *
+from .kern import *
+from .glyf import *
+from .fvar import *
+from .stat import *
+from .layout import *
+
 profile_imports = (
     (
-        ".",
```

Comment:
I don't quite understand how profile_imports works (why do we even have our own system for importing things?!) but can't we change just this to `.opentype` and leave the rest?

`*` imports are a bad idea (as the linter is telling you) because they make the namespace unpredictable. It's fine for something like this where we're not really doing any code, but in general you don't want adding a utility function to module `B` to suddenly add a new defined symbol to module `A`.

Reply:
It is part of the "magic" behaviour of inferring conditions and args to checks.

I have been actually working on a refactor of the profile to not use those profile_import 
statements, but by doing so, I think I started to grasp why they're there.

I'm pushing an updated set of commits now

Reply:
yes, I agree about the `import *` being usually a bad idea.